share/completions/connmanctl.fish

```diff
@@ -0,0 +1,40 @@
+function __fish_print_connman_services
+    connmanctl services | string replace -r '.* (.*)' '$1'
+end
+
+function __fish_print_connman_technologies
+    connmanctl technologies | string match '*Type*' | string replace -r '  Type = (.*)' '$1'
+end
+
+function __fish_print_connman_vpnconnections
+    connmanctl vpnconnections | string replace -r '.* (.*)' '$1'
+end
+
+complete -f -c connmanctl -n "test (count (commandline -opc)) -lt 2" -a "state" -d "Shows if the system is online or offline"
```

Comment:
Usually you'll want a function that prints the currently used command.

In the simple case this can be `__fish_seen_subcommand_from $commands`, so you'd do 

`complete -f -c connmanctl -n "not __fish_seen_subcommand_from state technologies clock [...]"`.

This is to support options before commands, which is frequently allowed.